cli/commands.go

```diff
@@ -2,44 +2,89 @@ package cli
 
 import (
 	"fmt"
-	"os"
+	"regexp"
 	"strings"
 
 	"github.com/rokka-io/rokka-go/rokka"
 )
 
+type UnknownCommandError string
+
+func (e UnknownCommandError) Error() string {
+	return string(e)
+}
+
 type Command struct {
-	args        []string
-	description string
-	fn          func(*rokka.Client)
+	Args        []string
+	Description string
+	fn          func(*rokka.Client, []string, map[string]string) error
 }
 
 var commands = []Command{
-	Command{[]string{"stackoptions", "list"}, "Show default stack options", func(c *rokka.Client) {
+	Command{[]string{"stackoptions", "list"}, "Show default stack options", func(c *rokka.Client, _ []string, _ map[string]string) error {
 		res, err := c.GetStackoptions()
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Error getting stack options: %s", err)
-			os.Exit(1)
+			return err
+		}
+		fmt.Println(PrettyJSON(res))
+
+		return nil
+	}},
+	Command{[]string{"organizations", "get", "<name>"}, "Get details of an organization", func(c *rokka.Client, args []string, _ map[string]string) error {
+		res, err := c.GetOrganization(args[0])
+		if err != nil {
+			return err
 		}
-		PrettyPrintJSON(res)
+		fmt.Println(PrettyJSON(res))
+
+		return nil
 	}},
 }
 
 func GetCommands() []Command {
 	return commands
 }
```

Comment:
let's just make `commands` exported instead of this useless getter.

Reply:
done